src/AbstractFixer.php

```diff
@@ -41,9 +41,9 @@
     protected $whitespacesConfig;
 
     /**
-     * @var FixerConfigurationResolver|null
+     * @var FixerConfigurationResolverInterface|null
      */
-    private $configurationDefinition;
+    protected $configurationDefinition;
```

Comment:
Not sure if we want to split `AbstractFixer`. If extracting configurable part, one could also want to extract whitespaceConfigurable part. but then we would end up in 2 new abstract classes, and then `header_comment` want to inherit both of them...
Please rethink that, if we really want to move that part away, we should do it fully, like moving `configure` into `AbstractConfigurableFixer`, this property and handling in constructor